cardano-cli/src/Cardano/CLI/Helper.hs

```diff
@@ -114,6 +118,10 @@ cborToTextByteString bs = do
   text <- cborToText bs
   pure $ LB.fromStrict $ Text.encodeUtf8 text
 
+cborToTextLazyByteString :: LB.ByteString -> ExceptT HelpersError IO LBS.ByteString
```

Comment:
Where is this used?

Reply:
I must have backed out the change that used it.

Reply:
I must have backed out the change that used it.